demos/projects/NXP/mimxrt1060/CMakeLists.txt

```diff
@@ -24,6 +34,7 @@ file(GLOB NXPCODE_SOURCES nxp_code/*.c nxp_code/lwip/*.c)
 set(PROJECT_SOURCES ${NXPCODE_SOURCES} main.c)
 
 add_executable(${PROJECT_NAME} ${PROJECT_SOURCES})
+add_executable(${PROJECT_NAME}-pnp ${PROJECT_SOURCES})
```

Comment:
can we group them like we did for other cmakes

<simple sample>
add_executable
add_target_include
add_target_link_libraries

<pnp sample>
add_executable
add_target_include
add_target_link_libraries